.gitlab-ci.yml

```diff
@@ -16,376 +16,25 @@ variables:
-    - cp -r ./Tooling_Release/* ./Tooling_Package/
-    - cp -r "./Documentation/Tooling for Web/" ./Tooling_Package/Documentation/
-  artifacts:
-    name: "TouchFree_Tooling_for_Web_${TF_TFW_VERSION}_${CI_COMMIT_SHORT_SHA}"
-    paths:
-      - ./Tooling_Package/*
-    expire_in: 2 weeks
-    when: always
-  tags:
-    - docker
-
-TF_Tooling::Unity::package:
-  stage: package
-  image: alpine:3.13
-  dependencies:
-    - TF_Tooling::Unity::build
-    - TF_Tooling::Docs::build
-  needs: [
-    "TF_Tooling::Unity::build",
-    "TF_Tooling::Docs::build"
-  ]
-  script:
-    - mkdir ./Tooling_Package/
-    - cp -r ./Tooling_Release/* ./Tooling_Package/
-    - cp -r "./Documentation/Tooling for Unity/" ./Tooling_Package/Documentation/
-  artifacts:
-    name: "TouchFree_Tooling_for_Unity_${TF_TFU_VERSION}_${CI_COMMIT_SHORT_SHA}"
-    paths:
-      - ./Tooling_Package/*
-    expire_in: 2 weeks
-    when: always
-  tags:
-    - docker
-
-TF_Service::package:
-  stage: package
-  dependencies:
-    - TF_Service::build
-    - TF_Service::Wrapper::build
-    - TF_Service::Tray::build
-  needs: [
-    "TF_Service::build",
-    "TF_Service::Wrapper::build",
-    "TF_Service::Tray::build"
-  ]
-  script:
-    # Package Service up neatly
-    - mkdir ./Service_Package/
-    - mkdir ./Service_Package/Licenses/
-    - mkdir ./Service_Package/Service/
-    - mkdir ./Service_Package/SettingsUI/
-    - mkdir ./Service_Package/Wrapper/
-    - mkdir ./Service_Package/Tray/
-    - cp -r ./Build/* ./Service_Package/Service/
-    - cp -r ./UIBuild/* ./Service_Package/SettingsUI/
-    - cp -r ./Service_Wrapper_Build/* ./Service_Package/Wrapper/
-    - cp -r ./Service_Tray_Build/* ./Service_Package/Tray/
-    - cp -r ./Third_Party_Licenses/* ./Service_Package/Licenses/
-
-  artifacts:
-    name: "TouchFree_Service_Package_${CI_COMMIT_SHORT_SHA}"
-    paths:
-      - ./Service_Package
-    expire_in: 2 weeks
-    when: always
-  tags:
-    - unity
-    - win10
-
-TouchFree::installer:
-  stage: installer
-  dependencies:
-    - TF_Service::package
-    - TouchFree::build
-  needs: [
-    "TF_Service::package",
-    "TouchFree::build"
-  ]
-  script:
-    # Run innosetup
-    - cd ./Scripts/
-    - "& ./compile_installer.bat ${env:TOUCHFREE_VERSION} $CI_PROJECT_DIR/TF_Service_Utilities/Installer/TouchFree_Installer.iss"
-
-    # Sign the installer with the UltraLeap certificate
-    - $cert=Get-ChildItem -Path Cert:\LocalMachine\My -CodeSigningCert
-    - Set-AuthenticodeSignature -FilePath "$CI_PROJECT_DIR/Installer_Build/TouchFree_${TOUCHFREE_VERSION}_Installer.exe" -TimestampServer 'http://timestamp.comodoca.com/authenticode' -Certificate $cert
+    - Set-AuthenticodeSignature -FilePath "$CI_PROJECT_DIR/Service_dotNet_Build/TouchFree Service.exe" -TimestampServer 'http://timestamp.comodoca.com/authenticode' -Certificate $cert
   artifacts:
-    name: "TouchFree_Installer_${TOUCHFREE_VERSION}_${CI_COMMIT_SHORT_SHA}"
+    name: "TouchFree_Service_dotNet"
```

Comment:
The build artefact contains two copies of the resulting exe, on in the root folder and one in the publish folder. I presume we don't really need both

Reply:
We do the same on develop for all artifacts, but we can change the 'copy' to a 'move' if we want I assume?